uppsrc/CtrlCore/GtkEvent.cpp

```diff
@@ -638,7 +638,7 @@ void Ctrl::Proc()
 			if(x)
 				kv = x->b;
 			else {
-				if(GetCtrl() | GetAlt()) {// fix Ctrl+Shift+1 etc...

```

Comment:
What about plain old ||?

Reply:
`GetCtrl()` and `GetAlt()` seems to have no side-effects, so short-cutting it with `||` should work well, and then it makes me wonder why it was not used already in original source.

Will change the PR.

Reply:
Very likely just harmless typo.